docs/quick_start.rst

```diff
@@ -0,0 +1,132 @@
+Quick Start
+===========
+
+Detect Action Potentials
+------------------------
+
+To detect action potentials, first construct three 1D numpy arrays: stimulus current ``i`` in pA,
```

Comment:
instead of "first construct", "we must provide" or something (I know the numpy array is technically being constructed ...).